src/main/webapp/src/app/map/map.component.ts

```diff
@@ -1,12 +1,34 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, AfterViewInit, ViewChild, ElementRef } from 
+  '@angular/core';
 
 @Component({
   selector: 'app-map',
   templateUrl: `map.component.html`,
   styleUrls: ['./map.component.css']
 })
 
-export class MapComponent implements OnInit {
-  constructor() {  }
-  ngOnInit() {  }
+export class MapComponent implements AfterViewInit {
+
+  @ViewChild("mapContainer", { static: false })
+  mapContainer!: ElementRef;
+  map!: google.maps.Map;
+  // Set initila location to Sydney coordinates.
+  lat = -33.865143;
+  lng = 154;
+  
+  // Coordinates to set the center of the map.
+  coordinates = new google.maps.LatLng(this.lat, this.lng);
```

Comment:
Nit: maybe have the variable as 'centerOfMapCoordinates' and remove the comment (just to have a self-explanatory variable)